ext/fs/std_fs.rs

```diff
@@ -61,7 +61,7 @@ impl FileSystem for RealFs {
       umask(Mode::from_bits_truncate(mask as mode_t))
     } else {
       // If no mask provided, we query the current. Requires two syscalls.
-      let prev = umask(Mode::from_bits_truncate(0o777));
+      let prev = umask(Mode::from_bits_truncate(0));
```

Comment:
This also aligns with node.js: https://github.com/nodejs/node/blob/94191f7b0984410dcb134faa96cd1a0741d5e8f0/src/node_process_methods.cc#L243